lib/shopify_api/resources/fulfillment_request.rb

```diff
@@ -0,0 +1,11 @@
+module ShopifyAPI
+  class FulfillmentRequest < Base
+    init_prefix :order
+
+    def order_id
+      @prefix_options[:order_id]
+    end
+
+    def cancel; load_attributes_from_response(post(:cancel, {}, only_id)); end
```

Comment:
Ugh, wrong gem I'm thinking of. I"m thinking of the `ActiveFulfillment::ShopifyApi` adapter. This ... is good.